realsense2_camera/package.xml

```diff
@@ -28,6 +28,9 @@
   <depend>tf2</depend>
   <depend>tf2_ros</depend>
   <depend>diagnostic_updater</depend>
+  <test_depend>ament_cmake_gtest</test_depend>
+  <test_depend>launch_testing</test_depend>
+  <test_depend>ament_cmake_pytest</test_depend>
```

Comment:
Maybe missing `launch_pytest`?

BTW, whats the difference between `launch_testing` and `launch_pytest`?

Reply:
@Nir-Az,  luanch_pytest is a relative new package that helps us filter tests based on markers. This is good for RS test grouping, as we can have test markers based on rosbag, devices (d435, d455 etc.). The package was already added as a dependency to the test.